utils/consolidateAllData.ts

```diff
@@ -1,88 +0,0 @@
-import formatGraph from './formatGraph'
```

Comment:
Coded this into the formatCountyData util because it was convenient at the time but I'm thinking that it might be better to keep this as its own util.